### DISCUSSION TITLE
Consider adding reticulum to another fdroid repository

### DISCUSSION THREAD

**Discussion by qwertyuiopzxcvbnmlkjhgfdsa1** [markqvist/Reticulum#478] [Ideas]
Hi,
have added Between the Borders repo in order to get fresh build of sideband. Currently the state of this repo is that it is not working. When i click to Update/Install Reticulum with fdroid, it says for second "Waiting to stats download..." and then it dies.

My idea which i would like to share with you is to switch to another repo, ideally which si installed by default on Fdroid to gain more users.

Have a nice day

**Comment by faragher** [comment]
Thanks for saying something. It was a permissions issue that is now fixed; I'm looking to fix the underlying deployment problem.

The issue with default F-Droid repos are there are only two: F-Droid and Guardian. While it may be worth investigating, I assumed being a part of Izzy On Droid was because there was a reason Sideband wasn't part of the main repo/Guardian. 

I'll look into it, but the in-house repo was a choice made fully aware of how it requires and extra step.

**Reply by qwertyuiopzxcvbnmlkjhgfdsa1** [reply]
Sideband downloading from repo is working now ! Thanks for fast fix.

**Reply by markqvist** [reply]
The reason that Sideband is not on F-Droid in the first place is that F-Droid needs to build all hosted packages from source on their own systems. I'm not even sure their build system can even support building Sideband, and even if it can, it will probably take some serious time setting it all up and maintaining it.

This is not something I can prioritize, or even have any real interest in, since I don't see it providing any significant and real benefit, compared to the amount of effort it requires. It's just as easy for a user to download the release APK directly, or just use the F-Droid repo provided by Between the Borders.

Furthermore, and as far as I know, the main F-Droid repository will not accept packages licensed under Creative Commons licenses, but I may be wrong here.

**Comment by LinuxinaBit** [comment]
I think it’s a good idea to keep Sideband on Izzyondroid just as a backup source if the BtB repo is inaccessible for some reason (and it could *technically* help with discovery too).
F-Droid and other clients know how to deal with two repos hosting the same app, so that shouldn’t be an issue either.

**Reply by faragher** [reply]
Agreed, but the BtB repo was created because it was no longer on Izzyondroid. My understanding it it was over using a CC license.

FDroid-Server as an application throws a non-stopping error if you don't use a license on the SPDX list (OSI or FSF: https://spdx.org/licenses/ ) so that may be the cause, or simply the owner of a service deciding what they do or don't want to host. That's fair, if disappointing.

In any case, if you know of any repos that handle compiled APKs, has an automated update system (I'm already going to lag behind on BtB), doesn't require in-house compilation, and is okay with CC licenses, please let me know.
